docs/readme.md

```diff
@@ -205,3 +210,18 @@ val read = for {
   value <- parsed.as[Meeting]
 } yield value
 ```
+### Akka
+
+```
```

Comment:
You can use a tag like ` ```scala mdoc` - then the example will be compiled when the readme is generated; this way the examples don't get out of date easily. You might then probably add the new project as a dependency to the readme sbt module.
If it gets too much, you can just remove the example and maybe add it later of course…

Reply:
@eikek Unfortunately the Akka example does not compile (the compiler does not see akka libs). I tried to resolve the issue but with no success. Perhaps you have an idea what is wrong?

Reply:
Oh, that's strange! Unfortunately, I have no idea. It looks good to me… 

Ah , just tested it here. It seems that the import `akka.bla` is not unique anymore. The compiler looks relative to the current package - there is also an `akka` package (from the module) and its also available from the root. I think you need to say `import _root_.akka.actor.typed.etc` to make it an "absolute path"

Reply:
I was not quite correct, it's not the current package, but an import earlier it imports `com.github.eikek.calev._` wihch also now makes the relative import `akka` available to the compiler. But you can leave it if you want, I'm going to create a release after merging anyways and will then update the readme again.

Reply:
Thanks for the hint. I fixed it!